Cargo.toml

```diff
@@ -109,7 +109,7 @@ fallible-iterator = "0.2"
 fallible-streaming-iterator = "0.1"
 memchr = "2.3"
 uuid = { version = "0.8", optional = true }
-smallvec = "1.0"
+smallvec = ">=1.6.1"
```

Comment:
This should just be written as "1.6.1" which implies >=1.6.1 <2.0.0, see [doc](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html)

Reply:
You are right.